charon/pkgs/maven.py

```diff
@@ -440,7 +450,12 @@ def handle_maven_del(
     )
     logger.info("Files deletion done\n")
 
-    # 4. Use changed GA to scan s3 for metadata refreshment
+    # 4. Delete related manifest from s3
+    logger.info("Start deleting manifest from s3")
+    s3_client.delete_manifest(prod_key, target, manifest_bucket_name)
+    logger.info("Manifest deletion is done\n")
```

Comment:
I think this step can be skipped with a log if manifest_bucket_name is not specified.

Reply:
@ligangty The case was already covered inside the delete_manifest method, others are sorted based on the comment.